src/transformers/models/idefics/processing_idefics.py

```diff
@@ -311,18 +325,55 @@ def __call__(
                 transforms.Normalize(mean=self.image_mean, std=self.image_std),
             ]
         )
-        inputs = processor(prompts, transform=image_transform, return_tensors="pt")
+        inputs = processor(text=prompts, transform=image_transform, return_tensors="pt")
         ```
 
         In order to help debug prompt generation enable `debug=True` which will show you what's happening.
 
         """
+        if images is None and text is None:
```

Comment:
we have a breaking change here no? `prompts` will be passed by previous workflows (ex: `prompts=xxx`), and we don't check if `prompt in kwargs`

Reply:
I think this should be handled by https://github.com/huggingface/transformers/blob/7a63c6f7c11ecdb423bedb5558b2cbe32c43ed37/src/transformers/models/idefics/processing_idefics.py#L236
with `prompts` being replaced by `text` automatically

Reply:
Ah right! missed this indeed, good to go then!